2022/0xxx/CVE-2022-0419.json

```diff
@@ -1,88 +1,89 @@
-                        ]
-                    },
-                    "vendor_name": "n/a"
+  "CVE_data_meta": {
+    "ASSIGNER": "security@huntr.dev",
+    "ID": "CVE-2022-0419",
+    "STATE": "PUBLIC",
+    "TITLE": "NULL Pointer Dereference in radareorg/radare2"
+  },
+  "affects": {
+    "vendor": {
+      "vendor_data": [
+        {
+          "product": {
+            "product_data": [
+              {
+                "product_name": "radareorg/radare2",
+                "version": {
+                  "version_data": [
+                    {
+                      "version_affected": "<",
+                      "version_value": "6.0.0"
+                    }
+                  ]
                 }
+              }
             ]
+          },
+          "vendor_name": "radareorg"
         }
-    },
-    "data_format": "MITRE",
-    "data_type": "CVE",
-    "data_version": "4.0",
-    "description": {
-        "description_data": [
-            {
-                "lang": "eng",
-                "value": "NULL Pointer Dereference in NPM radare2.js prior to 6.0.0."
-            }
-        ]
-    },
-    "impact": {
-        "cvss": {
-            "attackComplexity": "LOW",
-            "attackVector": "LOCAL",
-            "availabilityImpact": "LOW",
-            "baseScore": 5.9,
-            "baseSeverity": "MEDIUM",
-            "confidentialityImpact": "LOW",
-            "integrityImpact": "LOW",
-            "privilegesRequired": "NONE",
-            "scope": "UNCHANGED",
-            "userInteraction": "NONE",
-            "vectorString": "CVSS:3.0/AV:L/AC:L/PR:N/UI:N/S:U/C:L/I:L/A:L",
-            "version": "3.0"
-        }
-    },
-    "problemtype": {
-        "problemtype_data": [
-            {
-                "description": [
-                    {
-                        "lang": "eng",
-                        "value": "n/a"
-                    }
-                ]
-            }
-        ]
-    },
-    "references": {
-        "reference_data": [
-            {
-                "name": "https://huntr.dev/bounties/1f84e79d-70e7-4b29-8b48-a108f81c89aa",
-                "refsource": "CONFIRM",
-                "url": "https://huntr.dev/bounties/1f84e79d-70e7-4b29-8b48-a108f81c89aa"
-            },
-            {
-                "name": "https://github.com/radareorg/radare2/commit/feaa4e7f7399c51ee6f52deb84dc3f795b4035d6",
-                "refsource": "MISC",
-                "url": "https://github.com/radareorg/radare2/commit/feaa4e7f7399c51ee6f52deb84dc3f795b4035d6"
-            }
-        ]
-    },
-    "source": {
-        "advisory": "1f84e79d-70e7-4b29-8b48-a108f81c89aa",
-        "discovery": "EXTERNAL"
+      ]
     }
-}
\ No newline at end of file
+  },
+  "data_format": "MITRE",
+  "data_type": "CVE",
+  "data_version": "4.0",
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "NULL Pointer Dereference in GitHub repository radareorg/radare2 prior to 6.0.0."
```

Comment:
Should be 5.6.0

Reply:
@trufae - please see https://github.com/CVEProject/cvelist/pull/4434